crates/experimentation_platform/src/api/experiments/handlers.rs

```diff
@@ -648,6 +660,29 @@ async fn ramp(
         ))
         .get_result(&mut conn)?;
 
+    let (_, config_version_id) = fetch_cac_config(&tenant, &data).await?;
+
+    let webhook_event = if new_traffic_percentage == 0
```

Comment:
if the new ramp is 0 and previous ramp is not 0, but something else like 10 or 20, in that case it should not be `ExperimentStarted`

Reply:
But if the previous ramp is not 0, then the previous experimentStatus won't be CREATED, it would be INPROGRESS